specification/versioning.md

```diff
@@ -77,3 +77,33 @@ until the end of that component’s existence:
   MUST be provided as the latest `PATCH` version for the latest `MINOR` version
   of the latest `MAJOR` and SHOULD NOT be provided for previous `PATCH` or
   `MINOR` releases.
+
+## Release life cycle management
+
+Each repository `MUST` use [GitHub releases mechanism](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) to create snapshot of components
+and features delivered to the users. If release is also distributed via other 
+distribution channels (e.g. CDN, Registries, AppStores) it `MUST` be described 
+according to following rules by any means available in the given software 
+distribution solution. 
+ 
+Each release `MUST` clearly state the life cycle 
+status and described as: Active, Latest, Deprecated, End of Support. 
+At minimum the required information should be included in [CHAANGELOG.md](https://github.com/signalfx/gdi-specification/blob/main/specification/templates/CHANGELOG.md)
+and equivalent mechanism provided by system used for distributing software. 
+
+If new version is released previously released versions `MUST` be evaluated 
+according to stability guarantees and marked as:
+- Latest release `SHOULD` be marked as **Latest** indicating that this is
+  version recommended to new and existing users
```

Comment:
Should this not be only applied for "stable" releases. E.g. beta/rc releases are usually not marked as latest.

Reply:
Good example - we should keep the latest at the stable level as we do not want to promote latest on experimental releases. The new release  release should be marked as experimental or experimental, supported 
Btw no I think about it - instead of "Active" we should probably use "Supported"

Reply:
So actually instead of 
-	*Latest*
-	*Active*
-	*Deprecated:* End of Support expected on 2024-04-01
-	*End of Support:* Support has ended on 2024-04-01

We could have 
-       "Supported"
-	*Deprecated:* End of Support expected on 2024-04-01
-	*End of Support:* Support has ended on 2024-04-01

Reply:
@pellared change introduced and committed to PR. Please take a look